internal/util/aws.go

```diff
@@ -40,6 +43,9 @@ func (u *AWSUtil) UploadDirectoryToS3(localPath string, bucket string, prefix st
 		if err != nil {
 			log.Fatalln("Unable to get relative path:", path, err)
 		}
+		/*
+		#nosec CWE-22: added disclaimer to function docs
```

Comment:
@joshmarsh can you explain how this works / what is does?

Reply:
Sorry, I though I added a comment explaining nosec, but I’m not seeing it anymore.

Adding a //#nosec comment causes gosec to skip the code that you commented. 

Here are the docs on it: https://github.com/securego/gosec/blob/master/README.md#annotating-code